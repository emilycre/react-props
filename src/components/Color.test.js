import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color Component', () => {
  it('Renders a color', () => {
    const wrapper = shallow(<Color />);
    expect(wrapper).toMatchSnapshot();
  });
});
