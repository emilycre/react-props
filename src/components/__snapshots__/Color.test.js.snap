// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Component Renders a color 1`] = `
<dl>
  <dt>
    Name:
  </dt>
  <dd />
  <dt>
    Hex:
  </dt>
  <dd />
  <dt>
    RGB:
  </dt>
  <dd>
    Red: 
  </dd>
  <dd>
    Green: 
  </dd>
  <dd>
    Blue: 
  </dd>
</dl>
`;
